balancer-js/examples/contracts/veBAL-proxy.ts

```diff
@@ -1,23 +1,22 @@
-import { BalancerSDK } from '../../src/modules/sdk.module';
-import { Network } from '../../src';
-import dotenv from 'dotenv';
+/**
+ * This example shows how to the adjusted veBAL balance from the active boost delegation contract
+ * 
+ * How to run:
+ * yarn run example examples/contracts/veBAL-proxy.ts
+ */
+import { BalancerSDK, Network } from '@balancer-labs/sdk';
 
-dotenv.config();
-
-const sdk = new BalancerSDK(
-{ 
-    network: Network.GOERLI, 
-    rpcUrl: `https://goerli.infura.io/v3/${process.env.INFURA}`
+const sdk = new BalancerSDK({ 
+  network: Network.GOERLI, 
+  rpcUrl: 'https://rpc.ankr.com/eth_goerli'
```

Comment:
It does look way better without the need for env vars 👏

Reply:
true, adding env vars makes it hard to just run out of the box as well